docker/redis/start-redis.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+# Start Redis in background and wait 1 second
```

Comment:
If we're borrowing this whole file from that Stack Overflow answer, we should give it some credit. Maybe a comment near the top of this file that links to the answer and mentions that we borrowed it from there?